### ISSUE TITLE
How can I get datas with ajax?

### ISSUE THREAD

**Issue by anardil** [fabianlindfors/multi.js#2]
Hello,

Nice plugin :).
It will be great if the plugin can take the datas from an URL with ajax.

thank you.

**Comment by fabianlindfors** [comment]
This is a great idea!
Definitely something I want to implement, keep an eye out. :)

Thank you!

**Comment by xenogew** [comment]
Due to this, will you also support options as array of data like `{data: [{id: '', label:''}, ...]}`?
I would appreciate if you do support.

Thank you.

**Comment by slimstat** [comment]
Has anyone had any luck implementing this? Or some kind of lazy loading of data? We have selects with possible thousands of records, and would love to use some kind of progressive loading feature.

**Comment by abelardolg** [comment]
In the head of my html:
`<link rel="stylesheet" href="/plugins/dual_listbox/dist/multi.min.css">`

In the script of my html:
`<script src="/plugins/dual_listbox/dist/multi.min.js"></script>`

By defining the headers of my two selects:
```
$('#retention' ).multi({ non_selected_header: 'Header1',
          selected_header: 'Header2'
        });
```
These two elements were created by using the above code:
![image](https://user-images.githubusercontent.com/22522883/171625762-303367d1-9f61-46e8-a3de-f5f4405dbd8d.png)

How could I refer to the left select by using jQuery if not id is provided?

Since Ajax is not implemented yet, I would like to dinamically load data into my right select.
How could I do it?

```
<div class="form-group">
                            <label class="col-sm-2 control-label">Cat.Retención</label>
                            <div class="col-sm-7">
                                {% if retention_categories is defined %}
                                    <select multiple="multiple" name="retention[]" id="retention">
                                        {% for retention in retention_categories %}
                                            <option value="{{retention.id}}">{{retention.name}}</option>
                                        {% endfor %}
                                    </select>
                                {% endif %}
                            </div>
                        </div>
```
